src/main/java/frc/robot/constants/JsonConstants.java

```diff
@@ -43,6 +43,15 @@ public static void loadConstants() {
     blueFieldLocations = BlueFieldLocations.synced.getObject();
     drivetrainConstants = DrivetrainConstants.synced.getObject();
     visionConstants = VisionConstants.synced.getObject();
+
+    try {
+      scoringSetpoints.fillVariableL4WristMap();
+    } catch (Exception e) {
+      e.printStackTrace();
+      System.out.println(
+          "WARNING: L4 variable wrist map constants were incorrectly defined (different numbers of distances vs. angles), using L4 setpoint as placeholder map");
+      scoringSetpoints.fillPlaceholderL4WristMap();
```

Comment:
If the map is defined incorrectly, I think the appropriate action is to turn off variable scoring rather than filling in the table. Reasoning: we can be in position with the wrist but not the correct distance away for the nominal setpoint if we fill the table out, and then we'll be missing shots all over the place